python/dgl/graphbolt/data_block.py

```diff
@@ -30,17 +30,14 @@ class DataBlock:
     no node types, 'node_type' should be None.
     """
 
-    edge_feature: List[Dict[Tuple[str, str], torch.Tensor]] = None
```

Comment:
This `List` is corresponding to layers, not edge type.

Reply:
then why `node_feature` is not a list?

Reply:
Because of id space. All nodes are shared the same id space, while edges owns its own id space per layer for multi-layer subgraph.